### PR TITLE
seed data

### PR DESCRIPTION
**Pull request by whereismyjetpack**
Creates a fake_solr_document model that creates a solr document using faker. rake task runs that 20 times and adds the documents into  solr. I'm not sure if this is a sound approach or not.. I'm thinking we can expand this when we start writing tests 

something like doc = `FakeSolrDocument.new(access_levels: ['open_access'])` to create an open access document, with faked out other data.

### CODE REVIEW

File: lib/tasks/solr.rake [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
     task load_fixtures: :environment do
-      # TODO refactor
 
-      docs = JSON.parse(File.open('spec/fixtures/current_fixtures.json').read)
-      Blacklight.default_index.connection.add(docs)
-      Blacklight.default_index.connection.commit
+      20.times do 
+        doc = FakeSolrDocument.new
+        Blacklight.default_index.connection.add(doc.doc)
+        Blacklight.default_index.connection.commit
```
Comment (banukutlu) [inline lib/tasks/solr.rake:12]: @whereismyjetpack wonder if we can commit once at the end? It does not matter with this many docs though

**Review by banukutlu — APPROVED** [review]
this looks good to me, I agree that we can expand from this if needed later. 👍🏻

File: lib/tasks/solr.rake (hunk 1 above)
Comment (whereismyjetpack) [reply lib/tasks/solr.rake:12]: That was my thought, wouldn't be too hard to move the commit out of the loop. Also we might want to do something like `bundle exec rake solr:load_fixtures[100]` to add more and then the commit will matter. thanks!

Comment (banukutlu) [reply lib/tasks/solr.rake:12]: oh I liked this `bundle exec rake solr:load_fixtures[100]` idea very much!

**Pull request merged**